features/activity/retry_on_error/feature.ts

```diff
@@ -0,0 +1,25 @@
+import { Context } from '@temporalio/activity';
+import { WorkflowFailedError } from '@temporalio/client';
+import { Feature } from '@temporalio/harness';
+import * as assert from 'assert';
+import { workflow } from './feature.workflow';
+
+export const feature = new Feature({
+  workflow,
+  activities: {
+    alwaysFailActivity,
+  },
+  checkResult: async (runner, handle) => {
+    try {
+      await runner.waitForRunResult(handle);
```

Comment:
Better to use assert.rejects https://nodejs.org/api/assert.html#assertrejectsasyncfn-error-message

Reply:
:+1: Will change

Reply:
Changed